azext_iot/tests/digitaltwins/test_dt_import_jobs_int.py

```diff
@@ -0,0 +1,177 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from knack.log import get_logger
+from . import DTLiveScenarioTest
+from . import generate_resource_id
+from time import sleep
+
+logger = get_logger(__name__)
+
+
+class TestDTImportJobs(DTLiveScenarioTest):
+    def __init__(self, test_case):
+        self.storage_cstring = None
+        super(TestDTImportJobs, self).__init__(test_case)
+
+    def test_dt_importjobs(self):
+        self.wait_for_capacity()
+
+        storage_account_id = self.cmd(
+            "storage account show -n '{}' -g '{}'".format(self.storage_account_name, self.rg)
+        ).get_output_in_json()["id"]
+        instance_name = generate_resource_id()
+        # @avagraw - New APIs are only supported in East US in private preview.
+        dt_region = "eastus"
+        create_output = self.cmd(
+            "dt create -n {} -g {} -l {} --assign-identity --scopes {} --role 'Storage Blob Data Contributor'".format(
+                instance_name,
+                self.rg,
+                dt_region,
+                storage_account_id
+            )
+        ).get_output_in_json()
+        self.track_instance(create_output)
+
+        self.cmd(
+            "dt role-assignment create -n {} -g {} --assignee {} --role '{}'".format(
+                instance_name, self.rg, self.current_user, self.role_map["owner"]
+            )
+        )
+        # Wait for RBAC to catch-up
+        sleep(60)
+
+        # Upload import data files to storage account
+        valid_import_data_file = "./import_data/bulk-models-twins-relationships-valid.ndjson"
+        self.cmd(
+            "storage blob upload --connection-string '{}' --container-name '{}' --file '{}' --overwrite".format(
+                self.storage_cstring, self.storage_container, valid_import_data_file
+            )
+        )
+        invalid_import_data_file = "./import_data/bulk-models-twins-relationships-invalid.ndjson"
```

Comment:
will this work if you run the tests from a different folder (say the root)

Reply:
I will update this to the CWD format, like we do in other tests